### ISSUE TITLE
False positive: eslint import resolvers

### ISSUE THREAD

**Issue by clar-cmp** [depcheck/depcheck#571]
We use `eslint-import-resolver-typescript` with the below config in `eslintConfig`:

```json
      "import/resolver": {
        "typescript": {
          "alwaysTryTypes": true
        }
      },
```

This shows up as an unused dependency when using the `eslint` special.

**Comment by AndyOGo** [comment]
I still get a false positive for `eslint-import-resolver-typescript` as being missed.
But it comes with a shared eslint config.

```sh
> depcheck

Missing dependencies
* eslint-import-resolver-typescript: ./package.json
```

```sh
$ npm ls depcheck

└── depcheck@1.4.3
```
